### ISSUE TITLE
Unable to get sync progress using secure websockets

### ISSUE THREAD

**Issue by CVJoint** [hyperledger/besu#5216]
### Description
As an operator, I want secure websockets so that I can connect to the ws api from another machine securely. 

I receive an error when connecting an SSV node to besu's websocket while using besu's ws auth, but not when I disable the feature in besu and instead add http auth using a reverse proxy.

### Acceptance Criteria
* Resolve (potential) error in websocket authentication implementation.

### Steps to Reproduce (Bug)
1. Enable ws-authentication and credentials file.
2. Attempt secure wss connection:
    - I'm able to connect to the client:
    `INFO    goeth/goETH.go:140      successfully connected to execution client`
    - Upon starting, SSV checks the EC's sync status and I receive the error:
    `WARN    metrics/health_check.go:39      execution client is not healthy, trying again in 1sec   {"errors": ["could not get eth1 node sync progress"]}`
3. Disable ws-authentication.
4. Add http auth (user/pwd) to traefik.
5. Attempt secure wss connection - works as normal.

**Expected behavior:**
Normal function.

**Actual behavior:**
Unable to check sync status on websockets with besu auth enabled. I haven't detected any issues with the http rpc auth.

**Frequency:**
100%

### Versions (Add all that apply)
* Software version: besu/v23.1.1/linux-x86_64/openjdk-java-17
* Docker Version: 23.0.1
* Consensus Client & Version if using Proof of Stake: SSV

**Comment by jflo** [comment]
Can you post the full config file being used to configure besu?

**Comment by CVJoint** [comment]
**docker-compose.yml**:
```yaml
version: "3.7"
services:

  eth1:
    user: 1001:1001
    hostname: eth1
    container_name: eth1
    image: hyperledger/besu:latest
    restart: always
    stop_grace_period: 1m
    networks:
      net:
        ipv4_address: 192.168.32.101
    ports:
      - "30304:30304/tcp" # p2p
      - "30304:30304/udp" # p2p
      - "8545:8545" # RPC port
#      - "127.0.0.1:8545:8545" # RPC port
#      - "127.0.0.1:8546:8546" # websocket port
#      - "127.0.0.1:8551:8551" # engine RPC port
    volumes:
      - /home/eth/eth/besu/config:/config
      - /home/eth/eth/besu/data:/data
      - /etc/timezone:/etc/timezone:ro
      - /etc/localtime:/etc/localtime:ro
    environment:
      - BESU_CONFIG_FILE=/config/config.toml
    logging:
      driver: "local"

networks:
  net:
    external: true
```

**config.toml**:
```
data-path="/data"
data-storage-format="BONSAI"
nat-method="DOCKER"
sync-mode="X_SNAP"

network="goerli"

logging="INFO"
Xplugin-rocksdb-high-spec-enabled=true

max-peers=25
fast-sync-min-peers=3

host-allowlist=["*",]

p2p-enabled=true
p2p-host="0.0.0.0"
p2p-port="30304"

rpc-http-enabled=true
rpc-http-api=["WEB3", "ETH", "NET"]
rpc-http-host="0.0.0.0"
rpc-http-port="8545"
rpc-http-cors-origins=["*",]
rpc-http-max-active-connections=65536
rpc-max-logs-range=65536

rpc-http-authentication-enabled=true
#rpc-http-authentication-jwt-public-key-file="/config/publicRSAKey.pem"
rpc-http-authentication-credentials-file="/config/auth.toml"
#rpc-ws-authentication-enabled=true
#rpc-ws-authentication-credentials-file="/config/auth.toml"

rpc-ws-enabled=true
rpc-ws-api=["WEB3", "ETH", "NET"]
rpc-ws-host="0.0.0.0"
rpc-ws-port="8546"

metrics-enabled=true
metrics-host="0.0.0.0"
metrics-port="9545"

engine-host-allowlist=["*",]
engine-jwt-secret="/config/jwtsecret.hex"
engine-rpc-port="8551"
```

**auth.toml**:
```
[Users.<user>]
password = "$2a$1..."
permissions=["*:*"]
privacyPublicKey="U7AN..."
```

**Comment by CVJoint** [comment]
I've gotten besu auth to work.

Here are the changes I made:
1) removing the `privacyPublicKey` from the auth file (this was configured wrong, and I didn't realize it was optional)
2) creating the authentication bearer token
3) adding the token header to curl request: `-H 'Authorization: Bearer <token>'`

I was able to run my curl command above as well as the curl command from the besu docs. After 5 mins the token does expire, but it's working for me.

After some additional testing, I was also able to connect using Omnia with the bearer token.

Is there a way to add this header to an http string, similarly to how I added the user/pass? My use case is connecting to Rocket Pool and SSV.
SSV for example would need something like:
```
eth1:
ETH1Addr: ws://<user>:<pass>@<IP>:8546
```

I suppose the proper answer should be to reach out to those protocols and ask if they will add the bearer token auth method if it's not already an option. I can look at reaching out to them.
